Makefile

```diff
@@ -872,13 +872,12 @@ DEFCONFIG = $(call qstrip,$(UK_DEFCONFIG))
 # We don't want to fully expand UK_DEFCONFIG here, so Kconfig will
 # recognize that if it's still at its default $(CONFIG_DIR)/defconfig
 COMMON_CONFIG_ENV = \
-	CC=$(CC)\
```

Comment:
Why is this no longer needed?

Reply:
It was already incorrect. The correct version would have to be `CC=$(HOSTCC)` because `HOSTCC` is the compiler intended for compiling host tools. I can check if `CC` is used by the sub build system for KConfig. If so, I'll replace this line with the corrected version.

Reply:
Actually, the correct `CC` setting is already handed over when we invoke `Make` of KConfig, so this environment variable is overwritten and unnecessary to set with `COMMON_CONFIG_ENV`.